CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Apple has deprecated the [Create an App Store Version Submission](https://develo
   - `confirm_review_submission`,
   - `create_review_submission`,
   - `create_review_submission_item`.
+- Use default values for all arguments in `Keychain.add_certificates` and `XcodeProject.use_profiles`
```

Comment:
Version 0.26.0 is already released and we cannot include those changes there any more. Instead Create a new section at the top 
```
Unreleased
-------------

**Development**
- <Your changes here>
```